Spanner/src/Instance.php

```diff
@@ -497,6 +532,170 @@ function (array $database) {
         );
     }
 
+    /**
+     * Lazily instantiate a backup object
+     *
+     * Example:
+     * ```
+     * $backup = $instance->backup('my-backup');
+     * ```
+     *
+     * @param string $name The backup name.
+     *
```

Comment:
Also need to include the $options param here.

Reply:
Renamed that optional param into `array $backup = []`, this param is meant for internal use only, for injecting backup metadata while listing backups (to safe user an extra `backup->reload()` call).

Similar to [this](https://github.com/googleapis/google-cloud-php/issues/2524) issue 
Same is done for listing instances as well